org_onebusaway_iphone_Prefix.pch

```diff
@@ -13,11 +13,11 @@
     #import "OBACommonV1.h"
     #import "OBACommon.h"
 
-    #import "TestFlight.h"
     #import "GAI.h"
     #import "GAIFields.h"
     #import "GAITracker.h"
     #import "GAIDictionaryBuilder.h"
+    #define TFLog NSLog
```

Comment:
I redefined it thusly because I was concerned about the weirdness of redefining NSLog to be a variation on NSLog, and I think the extra logging info present in the redefinition of NSLog is actually quite useful. Always having the line number and `__PRETTY_FUNCTION__` available is quite helpful.